### PR TITLE
Add ಠ_ಠ as a \hmm latex completion.

### PR DESCRIPTION
**Pull request by Carreau**
See https://twitter.com/jakevdp/status/1239979963920445441 for background.

-- 

hmm

### CODE REVIEW

**Comment by Carreau** [comment]
Note that  the `ಠ` can be obtain with `\KANNADA LETTER TTHA<tab>` already, as many other valid letters.

**Comment by djkalmyk** [comment]
merge it, it's beautiful!

**Comment by Carreau** [comment]
I need to document it, test it, and it may end up its own completer provide instead of hacking the latex completer.

File: IPython/core/latex_symbols.py [hunk 1]
```diff
@@ -1295,6 +1296,7 @@
     "\\ttnine" : "𝟿",
     "\\underbar" : "̲",
     "\\underleftrightarrow" : "͍",
+    "\hmm" : "ಠ_ಠ",
```
Comment (ferrine) [inline IPython/core/latex_symbols.py:1299]: Need to be escaped
```suggestion
    "\\hmm" : "ಠ_ಠ",
```

Comment (ferrine) [reply IPython/core/latex_symbols.py:1299]: Is it?

Comment (Carreau) [reply IPython/core/latex_symbols.py]: Yes thanks !

**Pull request closed without merging**